services/src/main/java/org/exoplatform/webconferencing/externalvisio/rest/ExternalVisioConnectorRest.java

```diff
@@ -76,10 +76,11 @@ public Response createExternalVisioConnector(@RequestBody(description = "Externa
   @Operation(summary = "Retrieves the list of external visio connectors", description = "Retrieves the list of external visio connectors for an authenticated user", method = "GET")
   @ApiResponses(value = { @ApiResponse(responseCode = "200", description = "Request fulfilled"),
       @ApiResponse(responseCode = "500", description = "Internal server error"), })
-  public Response getExternalVisioConnectors(@Context
-  Request request) {
+  public Response getExternalVisioConnectors(@Parameter(description = "filter ExternalVisioConnector by status", required = true)
```

Comment:
Could we have a default value for enabled parameter ?
For example, if the parameter is not provided, return enabled providers